### PR TITLE
Add manifests to version root for simpler URLs

### PR DESCRIPTION
**Pull request by caseydavenport**
This shortens and standardizes the URLs for manifests for each release. I've just copied over the yamls into a new location, leaving the old URLs intact for now. This will give us a grace period before we remove the old ones.

They are both generated from the same templates so any changes will show up in both places. 

For example, these URLs in the current docs:

- https://docs.projectcalico.org/master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml
- https://docs.projectcalico.org/master/getting-started/kubernetes/installation/hosted/calico.yaml
- https://docs.projectcalico.org/master/getting-started/kubernetes/installation/hosted/canal/canal.yaml
- https://docs.projectcalico.org/master/getting-started/kubernetes/installation/hosted/canal/canal-etcd.yaml

would become these URLs after this PR:

- https://docs.projectcalico.org/master/manifests/calico.yaml
- https://docs.projectcalico.org/master/manifests/calico-etcd.yaml
- https://docs.projectcalico.org/master/manifests/canal.yaml
- https://docs.projectcalico.org/master/manifests/canal-etcd.yaml

- [ ] Tests
- [ ] Documentation
- [ ] Release note

```release-note
Simplify Kubernetes manifest URL locations. Existing URLs have been left intact, but will be removed in a future release. Please update any tooling to use the new manifest URLs.
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit c346529450390b883aa26fe01ae4a368a5af7224

https://deploy-preview-2437--calico.netlify.com

**Comment by caseydavenport** [comment]
Still need to do ALP manifests.

Might want to keep some sort of simple hierarchy given the number of manifests. 

e.g.:

- https://docs.projectcalico.org/alp/calico.yaml

**Comment by fasaxc** [comment]
Some of the manifests seem bigger than I was expecting (i.e. a full inline yaml rather than an include).  If we're going to have these in two places can you make them includes of a shared file instead?

File: master/manifests/calico-etcd.yaml [hunk 1]
```diff
@@ -4,5 +4,4 @@ layout: null
 {% helm %}
```
Comment (caseydavenport) [inline master/manifests/calico-etcd.yaml]: This diff is confusing - I didn't move these, I just deleted the unused ALP manifests.

**Pull request merged**